test_elasticsearch_async/test_connection.py

```diff
@@ -58,7 +58,7 @@ def test_error_is_properly_logged(connection, caplog, port, server):
 def test_timeout_is_properly_raised(connection, server):
     @asyncio.coroutine
     def slow_request():
-        yield from asyncio.sleep(0.01)
+        yield from asyncio.sleep(1)
```

Comment:
that this change is needed is very suspicious, the sleep is 2 orders of magnitude larger than the timeout and it still doesn't work - something is wrong here - maybe the timeout handling in connection that this is testing.